glide-core/src/protobuf/redis_request.proto

```diff
@@ -164,9 +162,9 @@ enum RequestType {
     GeoHash = 122;
     ObjectEncoding = 123;
     SDiff = 124;
-    ObjectIdletime = 125;
-    ObjectRefcount = 126;
-    LOLWUT = 100500;
+    ObjectIdleTime = 125;
+    ObjectRefCount = 126;
+    Lolwut = 100500;
```

Comment:
LOLWUT
why is it == 100500????

Reply:
A PR which assigns proper numbers for all (ALL) existing redis commands is coming soon after couple of renames. You can see the draft in https://github.com/Bit-Quill/glide-for-redis/pull/220